static/app/types/profiling.d.ts

```diff
@@ -1,4 +1,5 @@
 declare namespace Profiling {
+  type Release = import('sentry/types').Release;
```

Comment:
TIL you can assign a type w/ and import

Reply:
I tried to actually import type from outside the namespace declaration and all hell broke loose and it stopped working. After a quick search it turns out this is the recommended way